code/builders/builder-vite/src/codegen-importfn-script.ts

```diff
@@ -1,7 +1,9 @@
-import { relative } from 'node:path';
-
 import type { Options } from 'storybook/internal/types';
 
+import { genDynamicImport, genImport, genObjectFromRawEntries } from 'knitwork';
+import { normalize, relative } from 'pathe';
```

Comment:
similar to above, how does this improve over the current combination of Node and Vite APIs, to justify the extra dependency?

Reply:
As far as I know, there is no easy way to normalize windows paths in Node to always use forward-slash, which is what vite needs to handle the import correctly.

Reply:
But isn't that what we did already with `normalizePath` from Vite at the old line 30-32 below?

https://github.com/storybookjs/storybook/pull/28941/files#diff-a4ade24974a8c7f082261c73f22be07ec54ad61a3f9683b20035955a50fa1a8bL30-L32

Reply:
Oh, I wasn't aware of the existence of that helper (at least not the active part of my brain). I still think it would make sense to just use pathe everywhere, because it makes you forget about the issue completely. Would also allow to get rid of the `slash` dependency, and the 2 or 3 independent implementations of path normalizing in storybook (https://github.com/search?q=repo%3Astorybookjs%2Fstorybook%20slash&type=code)

Reply:
I see what you mean.
So what you're saying is that using `pathe` _everywhere_ is mentally easier/safer for us than using `node:path` + `slash` everywhere?

We've sort of been going in the opposite direction recetly, eg. replacing `fs-extra` with `node:fs` and a handful of manual logic, to slim down everything and make use of the modern APIs Node gives us today.

However when prebundled, `pathe` vs `slash` is 1.68 KB vs 98 B, which is an abysmal difference TBH.

Reply:
> I see what you mean. So what you're saying is that using `pathe` _everywhere_ is mentally easier/safer for us than using `node:path` + `slash` everywhere?

Yes. Additionally, pathe handles all kind of slashes also as input well (eg when calling `resolve` etc). So you don't need to worry about converting them to the correct format before calling the methods from `path`. `slash` is only this tiny bit of pathe: https://github.com/unjs/pathe/blob/main/src/_internal.ts

Reply:
Just had a look at the vite implementation, and it looks like they also just replace the slash: https://github.com/vitejs/vite/blob/5cac0544dca2764f0114aac38e9922a0c13d7ef4/packages/vite/src/shared/utils.ts#L27-L29